Jenkinsfile

```diff
@@ -0,0 +1,128 @@
+#!groovy
+
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+
+PLUGIN_NAME = "sample-angular-app"
+PLUGIN_ID = 'org.zowe.zlux.sample-angular-app'
+
+GITHUB_SSH_KEY = "zlux-jenkins" // This is required for git+ssh npm dependencies
+
+
+node("zlux-agent") {
+
+  def lib = library("jenkins-library").org.zowe.jenkins_shared_library
```

Comment:
We can use `library("jenkins-library@staging")` to temporarily import the latest updates on library. The `staging` branch has the changes of https://github.com/zowe/jenkins-library/issues/14, https://github.com/zowe/jenkins-library/issues/16, https://github.com/zowe/jenkins-library/issues/17.

Reply:
Library changes on staging have been merged into master. `@staging` can be removed now. Thanks for testing.